### PR TITLE
add code for "disabled by sift states"

### PR DESCRIPTION
**Pull request by jpicht**
## Coldbrew
New code to signal computation is disabled by sift states

### CODE REVIEW

File: tools.go
Comment (dmotylev) [inline tools.go:1]: We put these as `internal/tooldeps/tooldeps.go` with the pragma `//go:build for_go_mod_tidy_only`. I think there was an article on go.dev promoting that, although I can't find it now. Regardless, I think it is better to align this with the rest of our repos

**Review by dmotylev — CHANGES_REQUESTED** [review]
LGTM; let's just align tools dependency declaration with our other repos

File: tools.go
Comment (jpicht) [reply tools.go:1]: Why do we deviate from the original pattern?

**Comment by jpicht** [comment]
I've removed the `tools.go` commit.

**Review by dmotylev — APPROVED** [review]

**Pull request merged**